package.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+LAMBDAS=$1
+
+npx webpack --env.lambdas=${LAMBDAS}
```

Comment:
Maybe add -p for prod?

Reply:
There's already `mode: production` in `webpack.config.js` - is `-p` any different?

Reply:
Probably the same thing! I might be thinking of NODE_ENV=production